examples/asr/speech_to_text_infer.py

```diff
@@ -46,7 +46,6 @@ def main():
         "--asr_model", type=str, default="QuartzNet15x5Base-En", required=True, help="Pass: 'QuartzNet15x5Base-En'",
     )
     parser.add_argument("--dataset", type=str, required=True, help="path to evaluation data")
-    parser.add_argument("--wer_target", type=float, default=None, help="used by test")
```

Comment:
Why is this being removed?

Reply:
It's not being used. `wer_tolerance` is the correct argument